src/components/utils/http.js

```diff
@@ -22,7 +22,8 @@ const errorMessage = (status) => {
   }
 }
 
-const handleError = (status, callback) => {
+const handleError = ({status, response}, callback) => {
+  console.error(status, response)
   const message = errorMessage(status)
   Tracker.sendError(message)
```

Comment:
Fine for 500 - although I think sending the status code would be enough - the message does not add much.

But in the case of 401 errors we are still swallowing the reason we denied permission.

You could argue that a 4XX should not be logged to sentry at all, but if we do log them then we should send as much information as possible - i.e. include the response